Makefile

```diff
@@ -138,30 +138,26 @@ generate-chart: crossplane
 # where the operator is deployed. See https://github.com/operator-framework/operator-lifecycle-manager/issues/1361
 # and https://github.com/operator-framework/operator-lifecycle-manager/issues/2039
 
-olm: $(HELM) $(OLMBUNDLE)
+olm.build: $(HELM) $(OLMBUNDLE)
 	@$(INFO) Generating OLM bundle
 	@$(HELM) -n upbound-system template $(HELM_CHARTS_DIR)/$(PACKAGE_NAME) --set upbound.controlPlane.permission=edit > $(WORK_DIR)/olm.yaml
 	@$(SED_CMD) 's|RELEASE-NAME|$(PROJECT_NAME)|g' $(WORK_DIR)/olm.yaml
 	@rm -rf $(OLM_DIR)/bundle
 	@cat $(WORK_DIR)/olm.yaml | $(OLMBUNDLE) --version $(HELM_CHART_VERSION) --chart-file-path $(HELM_CHARTS_DIR)/$(PACKAGE_NAME)/Chart.yaml --extra-resources-dir $(CRDS_DIR) --output-dir $(OLM_DIR)
 
-helm.prepare: generate-chart
+olm.artifacts: olm.build
+	@mkdir -p $(abspath $(OUTPUT_DIR)/olm)
+	@tar -czvf $(abspath $(OUTPUT_DIR)/olm/$(VERSION)).tar.gz  -C $(OLM_DIR)/bundle .
 
-generate: go.vendor go.generate helm.prepare olm
-	@$(OK) Finished generating
+build.artifacts: olm.artifacts
 
-# Ensure a PR is ready for review.
-reviewable: generate lint
+helm.prepare: generate-chart
```

Comment:
I'm getting the following error when `helm.prepare.universal-crossplane` is not overridden. How does the CI pass?
```
==> Linting /Users/monus/go/src/github.com/upbound/universal-crossplane/cluster/charts/universal-crossplane
[ERROR] values.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 9: found character that cannot start any token
[ERROR] templates/: cannot load values.yaml: error converting YAML to JSON: yaml: line 9: found character that cannot start any token
[ERROR] : unable to load chart
	cannot load values.yaml: error converting YAML to JSON: yaml: line 9: found character that cannot start any token

Error: 1 chart(s) linted, 1 chart(s) failed
make[1]: *** [build/makelib/helm.mk:109: helm.lint.universal-crossplane] Error 1
make: *** [build/makelib/common.mk:425: reviewable] Error 2
```

Reply:
@muvaf are you on this branch? It is passing locally for me.

Reply:
Yes. I just did a fresh clone of the repo, pulled the PR via `gh pr checkout 142` and `make reviewable` fails because `values.yaml` still has the placeholder strings like `%%BOOTSTRAPPER_TAG%%`

Reply:
@muvaf okay I think I figured this out, it appears that running `make reviewable` does fail because those version env vars are not loaded, but when we run `make build` they are, which I why CI is passing. This looks like it might have happened because of the order https://github.com/upbound/universal-crossplane/pull/139 and #138. I'll open a fix to `main` and then add it here :+1:

Reply:
But [`make check-diff`](https://github.com/upbound/universal-crossplane/pull/142/checks?check_run_id=2821824684) does run `make reviewable`, right? Shouldn't it have failed?

Reply:
@muvaf it appears this behavior was recently changed in https://github.com/upbound/build/pull/156/commits/22101ce93c46c5c06bf998ea27dba778908edcd5

Reply:
Got it, thank you for the pointer! Opened https://github.com/upbound/universal-crossplane/issues/151